examples/extract_data/src/README.md

```diff
@@ -0,0 +1,26 @@
+# Quick setup
+
```

Comment:
Add an Overview section with a short description

```
## Overview

This example demonstrates how to use Airtop to extract data from a website using a prompt. By leveraging Airtop’s live view capabilities, you can have your users log into any of their accounts inside a browser session to provide your agents access to content that requires authentication. Airtop profiles can be used to persist a user’s login state across sessions and avoid the need to have them log in again.
```

That is what I meant on this [comment](https://github.com/airtop-ai/examples-python/pull/5#discussion_r1866662330). I just grabbed the first paragraph you had there and change `recipe` for example.

Reply:
Added